packages/webapp/src/components/Menu/ThreeDotsMenu/index.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import { forwardRef, ReactNode } from 'react';
+import clsx from 'clsx';
+import { BsThreeDots } from 'react-icons/bs';
+import FloatingMenu from '../FloatingButtonMenu/FloatingMenu';
+import DropdownButton from '../../Form/DropDownButton';
+import styles from './styles.module.scss';
+
+export type ThreeDotsMenuProps = {
```

Comment:
Fun fact, the three horizontal dots menu is also called "meatballs menu" (not saying we should change the name though 😂)

Reply:
I just googled and learned other menus have names too 😂
It sounds cute, so I renamed it🍴